synapse/lib/cell.py

```diff
@@ -4420,6 +4440,99 @@ async def ps(self, user):
 
         return retn
 
+    async def getAhaProxy(self, timeout=None, feats=None):
+
+        if self.ahaclient is None:
+            return
+
+        proxy = await self.ahaclient.proxy(timeout=timeout)
+        if proxy is None:
+            logger.warning('AHA client connection failed.')
+            return
+
+        if feats is not None:
+            for name, vers in feats:
+                if not proxy._hasTeleFeat(name, vers):
```

Comment:
Do we want to handle the case where AHA hasn't yet been updated and `_hasTeleFeat` isn't available or is the NoSuchMeth exception sufficient?

Reply:
The `_hasTeleFeat()` API is on the client telepath proxy, so if it knows to call it, it's there :)